src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -3,127 +3,101 @@
-
-register_cli_argument('sf cluster reliability', 'reliability_level',
-                      help='durability level.', **enum_choice_list(['Bronze', 'Silver', 'Gold']))
-
-register_cli_argument('sf cluster reliability', 'auto_add_node',
-                      help='Add node count automatically when changing reliability.')
-
-register_cli_argument('sf cluster setting set', 'settings_section_description', type=get_json_object,
-                      help='JSON encoded parameters configuration. Use @{file} to load from a file.'
-                      """for example: [{"section": "NamingService","parameter": "MaxOperationTimeout","value": 1000},{"section": "MaxFileOperationTimeout","parameter": "Max2","value": 1000]""")  # pylint: disable=line-too-long
-
-register_cli_argument('sf cluster setting remove', 'settings_section_description', type=get_json_object,
-                      help='JSON encoded parameters configuration. Use @{file} to load from a file.'
-                      """for example: [{"section": "NamingService","parameter": "MaxOperationTimeout"}]""")
-
-register_cli_argument('sf cluster client-certificate remove', 'client_certificate_common_names', type=get_json_object,
-                      help='JSON encoded parameters configuration. Use @{file} to load from a file.'
-                      """for example: [{"certificateCommonName": "test.com","certificateIssuerThumbprint": "22B4AE296B504E512DF880A77A2CAE20200FF922"}]""")
-
-register_cli_argument('sf cluster client-certificate add', 'client_certificate_common_names', type=get_json_object,
-                      help='JSON encoded parameters configuration. Use @{file} to load from a file.'
-                      """for example: [{"isAdmin":true, "certificateCommonName": "test.com",
-                         "certificateIssuerThumbprint": "22B4AE296B504E512DF880A77A2CAE20200FF922"}]""")
+
+def load_arguments(self, _):  # pylint: disable=too-many-statements
+    with self.argument_context('sf') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None, help='The resource group name')
+        c.argument('cluster_name', options_list=['--name', '--cluster-name', '-n'], help='Specify the name of the cluster, if not given it will be same as resource group name')
+        c.argument('location', validator=get_default_location_from_resource_group)
+        c.argument('secret_identifier', options_list=['--secret-identifier'], help='The existing Azure key vault secret URL')
+        c.argument('certificate_file', options_list=['--certificate-file'], help='The existing certificate file path for the primary cluster certificate.')
+        c.argument('parameter_file', options_list=['--parameter-file'], help='The path to the template parameter file.')
+        c.argument('template_file', options_list=['--template-file'], help='The path to the template file.')
+        c.argument('vm_password', options_list=['--vm-password'], help='The password of the Vm')
```

Comment:
Most of these options_lists can be removed as they are exactly what you'd get from the dest name conversion.

Reply:
true. fixed.